### PR TITLE
Merchant dashboard

### PR DESCRIPTION
**Pull request by GeorgieGirl24**
Testing:
  *Various Tests: add merchant_id to many users

MCV:
   
Routes:
   * Add routes to admin/merchant
Migration:
  * Add a merchant_id to users
  * Added a active? column to merchant

### CODE REVIEW

File: spec/features/admin/merchants/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    click_link "Log In"
+    fill_in("Email Address", with: "#{@user_3.email_address}")
+    fill_in("Password", with: "#{@user_3.password}")
+    click_button("Submit")
+  end
+
+  it 'can click on any merchants name and be routed to that unique merchants dashboard page and I can do all the functions of a merchant' do
+    visit '/admin/merchants'
+# save_and_open_page
```
Comment (hopesgit) [inline spec/features/admin/merchants/show_spec.rb]: Gotta get rid of this...

File: spec/features/admin/merchants/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    expect(page).to have_link("#{@bike_shop.name}")
+    click_on "#{@bike_shop.name}"
+
+    expect(current_path).to eq("/admin/merchants/#{@bike_shop.id}")
+    # expect(current_path).to eq(admin_merchant_path(@bike_shop.id))
```
Comment (hopesgit) [inline spec/features/admin/merchants/show_spec.rb:47]: and this

File: spec/features/admin/merchants/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    expect(page).to have_content(@order_1.id)
+    # expect(page).to have_content(@pencil.name)
```
Comment (hopesgit) [inline spec/features/admin/merchants/show_spec.rb:50]: and this

**Comment by ShaunDaneJames** [comment]
Thank you for all your hard work! Looks great!

**Review by hopesgit — COMMENTED** [review]
Nice job, George!

**Review by Todd-Estes — APPROVED** [review]
Lets get the money cream

**Pull request merged**